css/styles.css

```diff
@@ -0,0 +1,102 @@
+body {
+  font-family: 'Short Stack', cursive;
+}
+
+button {
+  font-family: 'Short Stack', cursive;
+}
+
+.orange {
+  background-color: #fecc90;
+}
+
+.home-header {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  justify-content: space-around;
```

Comment:
Only one of these is going to be applied, which is currently the last one. Remove the first one